manual/src/cmds/tail-mod-cons.etex

```diff
@@ -150,6 +158,31 @@ tail-mod-cons are as follows:
   using a convenient monadic notation.
 \end{itemize}
 
+\paragraph{Note: OCaml call stack size} Under OCaml 4, bytecode
+programs respect the "stack_limit" runtime parameter configuration
+(as set using "Gc.set" in the example above), or the "l" setting of
+the "OCAMLRUNPARAM" variable. Native programs ignore these settings
+and only respect the operating system native stack limit, as set by
+"ulimit" on Unix systems. Most operating systems run with a relatively
+low stack size limit by default, so Stack Overflow on
+non-tail-recursive functions are a common programming bug.
+
+Under OCaml 5, native code does not use the native system stack for
+OCaml function calls anymore, so it is not affected by the operating
+system native stack size; both native and bytecode programs respect
+the OCaml runtime's own limit. The runtime limit is set to a much
+higher default than most operating system native stacks, so we expect
```

Comment:
Who is that `we`? I would rather use a passive form to emphasize the `Stack Overflow occurences` subject.